File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/29a_compito_java_20190411/Trainer.py
from Dipendente import Dipendente


class Trainer(Dipendente):
    def __init__(self, codice, tipo, nome, ore_settimanali, costo, specialita):
        super().__init__(codice, tipo, nome, costo)
        self._ore_settimanali = ore_settimanali
        self._specialita = specialita

    def __str__(self):
        return super().__str__() + "\ttrainer\t" + str(self._ore_settimanali) + "\t" + self._specialita + "\t-\t-\t-\t" +\
            str(self._costo)

File: II - Linguaggi di programmazione e laboratorio/Esercizi/Java/34_20221221/29-soluzione-a/Java/Trainer.java
import java.util.*;

public class Trainer extends Dipendente {
	
  private int ore;
  private String spec;
  
  public Trainer(int codice,  String nome, int ore, String spec, double costo)
  {
    super( codice, nome, "trainer",costo);
     this.ore=ore;
     this.spec=spec;
  }
    	public String toString(){
		return super.toString()+"\t"+ore+"\t"+spec+"\t-\t-\t-\t"+getCosto();
	}


}
